src/grpc.js

```diff
@@ -124,10 +126,35 @@ class LndGrpc extends EventEmitter {
       this.emit('tor.started')
     }
 
-    // Probe the services to determine the wallet state.
-    const walletState = await this.determineWalletState()
+    // For lnd >= 0.13.*, the state service is available which provides with
+    // the wallet state. For lower version of lnd continue to use WalletUnlocker
+    // error codes
+    let walletState
+    this.checkRPCStatus = false
```

Comment:
I think a better name would be like `isStateServiceAvailable`

Reply:
Done.